src/main/java/com/epam/test_generator/services/AdminService.java

```diff
@@ -21,26 +21,26 @@
     private RoleService roleService;
 
     /**
-     * Changes user role to role specified in changeUserRoleDTO.
+     * Changes user role to role specified in updateUserRoleDTO.
      * Searching user occurs by e-mail address.
-     * @param changeUserRoleDTO
+     * @param updateUserRoleDTO
      */
-    public void changeUserRole(ChangeUserRoleDTO changeUserRoleDTO) {
+    public void changeUserRole(UpdateUserRoleDTO updateUserRoleDTO) {
```

Comment:
Can we generify user-update operations so client would use UserUpdateDTO for all update operations?